### ISSUE TITLE
Some glyphs are not rendered correctly on a PowerShell prompt

### ISSUE THREAD

**Issue by Tragetaschen** [microsoft/terminal#1958]
# Environment

```none
Windows build number: 10.0.18932.1000, but I'm seeing at least since 17763
Windows Terminal version (if applicable): 0.2.1831.0
```

# Steps to reproduce
Open a PowerShell and type `123¹²³` on the prompt (or paste it)

# Expected behavior
It should show up as `123¹²³`

# Actual behavior
It is rendered as if had typed `1231²3`, but only on the promt, not in error messages

![image](https://user-images.githubusercontent.com/1947968/61175209-a784ce00-a5ab-11e9-84cc-a1b7991df18a.png)

**Comment by sunyinqi0508** [comment]
This seems like a bug from PowerShell that substitutes ¹ with 1 rather than wt.

**Comment by DHowett-MSFT** [comment]
Good guess! Just checked the powershell parser, though. [These](https://github.com/PowerShell/PowerShell/blob/d6390461b2eeafb8aeb5dcfeff8134c2bca90d86/src/System.Management.Automation/engine/parser/CharTraits.cs#L6-L25) are the only special characters it treats differently.

Incidentally, though, you mentioned that you've been seeing it since 17763 -- so it's definitely not going to be a Windows Terminal issue. I can't even paste `123¹²³` in through conhost. PSReadline drops the superscripts completely.

If I `Remove-Module PSReadline`, it suddenly starts working.

It might be worth following up at [lzybkr/PSReadline](https://github.com/lzybkr/PSReadline). 17763 is the first version that included PSReadline 2.0.0.
